src/components/Nps/index.test.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+import { act, render, screen, within } from '@testing-library/react';
+import { sequence } from '@travelperksl/fabricator';
+
+import { answerFabricator } from 'tests/fabricator';
+import { Answer } from 'types/answer';
+
+import Nps, { npsDataTestIds } from '.';
+
+describe('Nps', () => {
+  const answers = answerFabricator.times(10, { text: () => sequence('npsScore').toString() });
+
+  it('renders a multi choice component', () => {
+    const onValuesChanged = () => {
+      // Do nothing
+    };
+
+    render(<Nps items={answers} onValuesChanged={onValuesChanged} />);
+
+    const nps = screen.getByTestId(npsDataTestIds.base);
+
+    expect(nps).toBeVisible();
+  });
+
+  describe('given the sixth score is clicked', () => {
+    it('renders the first six components as selected state', () => {
+      let selectedValues: Answer[] = [];
+
+      const handleValuesChanged = (answers: Answer[]) => {
+        selectedValues = answers;
+      };
+
+      render(<Nps items={answers} onValuesChanged={(answers) => handleValuesChanged(answers)} />);
+
+      const nps = screen.getByTestId(npsDataTestIds.base);
+
+      act(() => {
+        within(nps).getByText(answers[5].text).click();
+      });
+
+      expect(within(nps).getByText(answers[0].text)).not.toHaveClass('opacity-50');
```

Comment:
Same comment as previous PR, what is the `opacity-50`?

Reply:
@hoangmirs added ✅